src/sgame/sg_missile.cpp

```diff
@@ -438,13 +445,17 @@ static void MissileImpact( gentity_t *ent, const trace2_t *trace )
 
 void G_ExplodeMissile( gentity_t *ent )
 {
+	if ( ent->s.modelindex == MIS_FIREBOMB )
+	{
+		G_FirebombMissileIgnite( ent );
+	}
+
```

Comment:
This is kinda hacky to special case MIS_FIREBOMB here? Is this just to avoid the boilerplate of defining a new component for this?

Reply:
Besides having a firebomb component (which seems fine as it does have its own specialized logic), I guess I'd need to add another CBSE message to allow the behavior to be specialized here. I'm a little loath to add extension points that don't have at least 2 users.